table/table_builder.h

```diff
@@ -56,7 +56,8 @@ struct TableBuilderOptions {
       CompressionType _compression_type,
       const CompressionOptions& _compression_opts,
       const std::string* _compression_dict, bool _skip_filters,
-      const std::string& _column_family_name, int _level)
+      const std::string& _column_family_name, int _level,
+      const uint64_t _creation_time = 0)
```

Comment:
Thanks for letting me know about your experience regarding custom table formats. I didn't even know that such a feature existed (other than a couple of table formats documented on the wiki) :) . Good to know so that I keep that mind for future work. I plan to defer this too until a need arises or someone asks for it.